spack/package.py

```diff
@@ -181,85 +201,53 @@ def setup_run_environment(self, env):
 
     @run_after("install")
     def check_install_version(self):
```

Comment:
This function is almost the same as `check_install_platform` and `check_install_stdprob3`. An additional helper function `_check_install(*oommf_args)` could avoid some code duplication.

Reply:
Good idea. That's done now.